src/GRA.Web/GRA.Web.csproj

```diff
@@ -46,6 +46,14 @@
     <None Remove="wwwroot\js\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="Areas\MissionControl\Views\DailyTips\Details.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Remove="Areas\MissionControl\Views\DailyTips\Index - Copy.cshtml" />
+  </ItemGroup>
+
```

Comment:
You shouldn't need any of these changes to the `.csproj` file. go ahead and remove these lines.